src/pages/editor/EditorDoi.tsx

```diff
@@ -13,7 +13,7 @@ export const EditorDoi = () => {
       <Helmet>
         <title>{t('common.doi_long')}</title>
       </Helmet>
-      <Typography id="doi-label" variant="h2" sx={{ mb: '2rem' }}>
+      <Typography id="doi-label" variant="h1" sx={{ mb: '2rem' }}>
```

Comment:
Denne har mer margin enn alle andre. Kanskje best å samkjøre alle til å bruke `gutterBottom` eller ev. `1rem`?
```suggestion
      <Typography id="doi-label" variant="h1" gutterBottom>
```